kpi/deployment_backends/mixin.py

```diff
@@ -75,6 +75,7 @@ def _mark_latest_version_as_deployed(self, save: bool = False):
         latest_version.deployed = True
         latest_version.save()
         self.date_deployed = latest_version.date_modified
+
```

Comment:
Probably a mistake to commit this newline.

Reply:
Indeed.